packages/insomnia-inso/src/cli.js

```diff
@@ -174,6 +174,7 @@ export function go(args?: Array<string>, exitOverride?: boolean): void {
     .option('-a, --appDataDir <dir>', 'set the app data directory')
```

Comment:
```suggestion
    .option('-a, --appDataDir <dir>', 'set the app data directory (deprecated; use --src instead')
```

I don't think it can be removed altogether otherwise the alias will not be recognized correctly.